README.md

```diff
@@ -12,7 +12,7 @@ Read our blog post in [Hacker Noon](https://medium.com/hackernoon/ironhide-bette
 
 ## Installation
 
-`npm install -g @ironcorelabs/ironhide`
+`npm install -g --unsafe-perm @ironcorelabs/ironhide`
```

Comment:
I don't think we should have this be the default install command. I don't know of anybody else who has run into this error and it looks pretty sketchy as an argument. I'd prefer we added a "Troubleshooting" section below with a "Install fails with `EACCES: permission denied, mkdir ...`" and have a suggestion to add this flag.

Reply:
Agreed.

Reply:
I've never had the install work, with fresh installs on at least 3 different clean systems. But I'll change the README.